chapters/1.md

```diff
@@ -1,100 +1,185 @@
 
-In order to help mitigate such risks, the traditional banking system can utilize a new technology called the blockchain. A blockchain is a distributed database that continuously adds to a record of all the transactions that have occurred within a network. This database is often referred to as a ledger, because the data contains a large list of transactions that have taken place.
+This process is shown in Figure X.1:
 
-The first distributed blockchain was conceptualized in 2008 by an anonymous person or group known as Satoshi Nakamoto and implemented in 2009 as a core component of Bitcoin, where it serves as the public ledger for all transactions. The invention of the blockchain made Bitcoin the first digital currency to solve the double spending problem without the need for a trusted authority or central server.
+[Figure X.1](soon)
 
-Within a distributed network, if you have many disparate parties with no central authority, how do you ensure they all agree? The blockchain is a way of maintaining consensus in a decentralised distributed network. Transactions are grouped into “blocks”, which are linked and secured using cryptography, and every time a new block is solved (i.e. checked, signed, and verified by network participants), these transactions are recorded on the blockchain in chronological order. What goes into a block is just data, but it can be anything. The most obvious example is financial transactions.
+There are a few drawbacks and risks to this system, and it requires total trust of banks. Maria, George, and the banks must act on faith that transactions are legitimate and that the ledgers are kept honestly. This trust in the intermediate third parties poses a risk, since the banks or a nefarious actor may create unlimited money by fraudulently editing the ledger balances or transaction database.
 
-Consensus within the network is maintained because every time a new block is solved (usually every 10 minutes or so), all participants in the network must agree on the state of the blockchain; When this consensus occurs, each network participant's copy of the blockchain syncs so they all match. That way all copies march in lockstep, even though there is no one central master copy. As a result, the blockchain is completely unalterable as a historical record. If you take your copy of the blockchain and go back and change something in an earlier block, your copy of the blockchain will simply be rejected in 10 minutes at the next re-sync because your copy would not match the rest. This provides security to the network and removes the need for a centralized trusted third party.
+Furthermore, Maria does not actually have possession of $977241, only an IOU from her bank that she must trust is redeemable. She has no way to audit her bank to verify whether they actually have $977241. 
```

Comment:
s/`$977241`/`of the money in her account`

s/`an IOU`/`a promise`
You initially pointed out that IOU wasn't a universal term. We probably should stray from using it still.

s/`actually have $977241.`/`actually have the amount of money that they say that they are holding for her.`

Reply:
I think it may be valuable then to use realistic values for balances in account then. Possibly $4500 or something close to an amount someone might actually realistically see in their bank on a day to day basis.